<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasuringWindow</class>
 <widget class="QMainWindow" name="MeasuringWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>659</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/MyResources.qrc">
    <normaloff>:/Images/icons/logo.jpg</normaloff>:/Images/icons/logo.jpg</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1084</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuF_jl">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_MDF_file"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuN_zet">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionSignal_window"/>
    <addaction name="actionGraphic_Window"/>
    <addaction name="actionAlign_windows"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_Signals"/>
    <addaction name="actionMDF_file_info"/>
   </widget>
   <widget class="QMenu" name="menuOthers">
    <property name="title">
     <string>Others</string>
    </property>
    <widget class="QMenu" name="menuProfile">
     <property name="title">
      <string>Profile</string>
     </property>
     <addaction name="actionSaveProfile"/>
     <addaction name="actionLoadProfile"/>
     <addaction name="separator"/>
     <addaction name="actionImportProfile"/>
     <addaction name="actionExportProfile"/>
     <addaction name="separator"/>
     <addaction name="actionCloseProfile"/>
    </widget>
    <addaction name="menuProfile"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuF_jl"/>
   <addaction name="menuN_zet"/>
   <addaction name="menuOthers"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSignal_window"/>
   <addaction name="actionGraphic_Window"/>
   <addaction name="actionAlign_windows"/>
  </widget>
  <action name="actionExportProfile">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionImportProfile">
   <property name="text">
    <string>Import</string>
   </property>
   <property name="toolTip">
    <string>Profil betöltése</string>
   </property>
  </action>
  <action name="actionCloseProfile">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Profil bezárása</string>
   </property>
  </action>
  <action name="actionSelect_Signals">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select Signals</string>
   </property>
  </action>
  <action name="actionSignal_window">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/MyResources.qrc">
     <normaloff>:/Images/icons/Window Left Panel.png</normaloff>:/Images/icons/Window Left Panel.png</iconset>
   </property>
   <property name="text">
    <string>Signal Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionGraphic_Window">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/MyResources.qrc">
     <normaloff>:/Images/icons/Window Performance.png</normaloff>:/Images/icons/Window Performance.png</iconset>
   </property>
   <property name="text">
    <string>Graphic Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionOpen_MDF_file">
   <property name="text">
    <string>Open MDF file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Alt+Q</string>
   </property>
  </action>
  <action name="actionMDF_file_info">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>MDF file info</string>
   </property>
  </action>
  <action name="actionAlign_windows">
   <property name="icon">
    <iconset resource="resources/MyResources.qrc">
     <normaloff>:/Images/icons/Cascade-512.png</normaloff>:/Images/icons/Cascade-512.png</iconset>
   </property>
   <property name="text">
    <string>Align windows</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSaveProfile">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoadProfile">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/MyResources.qrc"/>
 </resources>
 <connections/>
</ui>
